src/util/util_vector.h

```diff
@@ -151,10 +151,17 @@ namespace dxvk {
 
   inline Vector4 replaceNaN(Vector4 a) {
     Vector4 result;
+#ifdef CPU_ARCH_X86
     __m128 value = _mm_load_ps(a.data);
     __m128 mask  = _mm_cmpeq_ps(value, value);
            value = _mm_and_ps(value, mask);
     _mm_store_ps(result.data, value);
+#else
+    if (a[0] != a[0]) a[0] = 0;
```

Comment:
0.0f

Why not do a loop here too?

Reply:
Fixed, I thought unrolled loop would be better